greedy-timetabling/src/main/java/org/acme/schooltimetabling/solver/GreedyService.java

```diff
@@ -0,0 +1,4 @@
+package org.acme.schooltimetabling.solver;
+
+public class GreedyService {
```

Comment:
empty, but you can inject this into TimeTableResource and have one method

TimeTable solve(TimeTable)

that has that greedy code

Reply:
Fixed in 58bdbc36